superset-frontend/src/assets/stylesheets/superset.less

```diff
@@ -545,7 +545,6 @@ td.filtered {
   display: flex;
   flex-direction: column;
   justify-content: center;
-  display: none;
```

Comment:
display flex is set in this class already, why are we adding display none here??? this seems obviously wrong, but i'll test in a test env before merge